BUILD.md

```diff
@@ -89,41 +121,30 @@ For other system requirements and other dependencies, please see [this section](
 |**Build Shared Library**|--build_shared_lib||
 |**Build Python wheel**|--build_wheel||
 |**Build C# and C packages**|--build_csharp||
-|**Build WindowsML**|--use_winml<br>--use_dml<br>--build_shared_lib|WindowsML depends on DirectML and the OnnxRuntime shared library.|
-|**Build Java package**|--build_java|Creates an onnxruntime4j.jar in the build directory, implies `--build_shared_lib`|
 
+---
 
-# Additional Build Instructions
-The complete list of build options can be found by running `./build.sh (or .\build.bat) --help`
+## Build ONNX Runtime Server on Linux
```

Comment:
Sorry, [this](https://github.com/microsoft/onnxruntime/blob/master/docs/Server.md) is the latest build doc for ORT Server.

Reply:
Thanks. Updated.